site/faq.md

```diff
@@ -15,7 +15,7 @@ the YAML manifests from `cloud.weave.works`.
 
 ## Disabling Scope Write Access
 
-Can be done by using the `probe.no-controls` option and set it to false for the scope agents. This can be done in the scope deployment manifest under the `weave-scope-agent`'s argument section with `—probe.no-controls=true`.
+Can be done by using the `probe.no-controls` option and set it to false for the scope agents. This can be done in the scope daemonset manifest under the `weave-scope-agent`'s argument section with `--probe.no-controls=true`.
```

Comment:
Nowadays we have both a Deployment (which handles Kubernetes data for the whole cluster) and a DaemonSet (which handles everything else on a per-host basis).

Reply:
Ah. Does it need to be passed on to both then? I only noticed one (my cluster is also a single node)

My update went to the daemonset and it seemed to do the trick

Reply:
I read once more carefully and it looks like there was another error in this paragraph. I think it reads correctly now.